.gitmodules

```diff
@@ -45,7 +45,7 @@
 [submodule "percona-toolkit"]
 	path = sources/percona-toolkit/src/github.com/percona/percona-toolkit
 	url = https://github.com/percona/percona-toolkit.git
-	branch = release-3.0.10
+	branch = 3.0
```

Comment:
Why don't use released version?

Reply:
Should I keep the old version? New explain is merged into 3.0

Reply:
In general, we don't want to depend on unreleased software. We would depend on git tags for released version if that would be possible with git modules. But since it is not possible, we should depend on release branches, not on trunk version, and 3.0 is a trunk, isn't it?

Reply:
Yes, it is a trunk branch. 
I'll change it to release-3.0.10 :+1:

Reply:
Then we will not use new explain code?

Reply:
Here? Don't know. How should I solve it? New PT release?

Reply:
Please work with @alexanderblinkov-percona on that.
We can't relay on unreleased dependencies unless we absolutely have to, and it doesn't look like we have to there